mortgage/mortgage_ibis.py

```diff
@@ -198,46 +200,88 @@ def etl_ibis(
+                    "Loading mortgage for ibis by Pandas not implemented yet"
+                )
+            elif import_mode == "fsi":
+                omnisci_server_worker._conn.create_table_from_csv(
+                    f"{table_prefix}_acq",
+                    f"{mb.acq_data_path}/Acquisition_{year}Q{quarter}.txt",
+                    acq_schema,
+                    database_name,
+                    delimiter=",",
+                    header="true",
+                    fragment_size=fragments_size[0],
+                )
+                etl_times["t_readcsv"] += timer() - t0
+                etl_times["t_connect"] += omnisci_server_worker.get_conn_creation_time()
+                # iterate perf files
+                perf_files = [
+                    f
+                    for f in Path(mb.perf_data_path).iterdir()
+                    if f.match(f"Performance_{year}Q{quarter}.txt*")
+                ]
+                for perf_file in perf_files:
+                    print(f"Loading {perf_file}")
+                    omnisci_server_worker._conn.create_table_from_csv(
+                        f"{table_prefix}_perf",
+                        str(perf_file),
+                        perf_schema,
+                        database_name,
+                        delimiter=",",
+                        header=True,
+                        fragment_size=fragments_size[1],
+                    )
+                    etl_times["t_readcsv"] += timer() - t0
+                    etl_times["t_connect"] += omnisci_server_worker.get_conn_creation_time()
+
+                    # Second connection - this is ibis's ipc connection for DML
+                    t0 = timer()
+                    omnisci_server_worker.connect_to_server(database_name, ipc=ipc_connection)
+                    acq_table = omnisci_server_worker.database(database_name).table(
+                        f"{table_prefix}_acq"
+                    )
+                    perf_table = omnisci_server_worker.database(database_name).table(
+                        f"{table_prefix}_perf"
+                    )
+                    etl_times["t_connect"] += timer() - t0
+
+                    t_etl_start = timer()
+                    df = run_ibis_workflow(acq_table, perf_table, leave_category_strings)
+                    dfs.append(df)
+                    etl_times["t_etl"] += timer() - t_etl_start
+                    # drop the performance table
+                    omnisci_server_worker.connect_to_server(database_name)
+                    omnisci_server_worker.drop_table(table_name=f"{table_prefix}_perf")
+                # drop the acquisition table
+                omnisci_server_worker.connect_to_server(database_name)
+                omnisci_server_worker.drop_table(table_name=f"{table_prefix}_acq")
+        import pandas as pd
+
+        ibis_df = dfs[0] if len(dfs) == 1 else pd.concat(dfs)
+    else:  # re-use existing tables for etl
```

Comment:
```suggestion
    else:  # use pre-populated tables for etl
```
after this PR is merged there would be no way to use the benchmark to fill the tables, so we should probably not "re-use" but "use pre-populated" tables